quickstart/101-virtual-network-manager-create-management-group-scope/main.tf

```diff
@@ -0,0 +1,166 @@
+# Create the Resource Group
+
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  location = var.resource_group_location
+  name     = random_pet.rg_name.id
```

Comment:
Would you please change the name to `"101-vnmcmgs-${random_pet.rg_name.id}"? Sometimes I found some legacy resources in our testing environment, and I have no clue which tests caused these pending resources, a name with such prefix could help me a lot, thanks!

Reply:
Done.